src/Client.php

```diff
@@ -261,43 +262,40 @@ public function userReport(string $report_uid,
     /**
      * {@inheritdoc}
      */
-    public function userReportMake(string $report_type_uid,
-                                   string $type,
-                                   string $value,
-                                   ?array $options = [],
-                                   ?bool $is_force = false,
-                                   ?string $on_update = null,
-                                   ?string $on_complete = null,
-                                   ?array $data = null): UserReportMakeResponse
+    public function userReportMake(ReportMakeParams $params): UserReportMakeResponse
     {
-        $request_options = [];
+        $request_body = [
+            'queryType' => $params->getType(),
+            'query'     => $params->getValue(),
+        ];
 
-        if (\is_bool($is_force)) {
-            $request_options['FORCE'] = $is_force;
-        }
+        $options = [
+            'FORCE' => $params->isForce(),
```

Comment:
`FORCE` was optional previously, now this optional will be passed into each request. This is correct?

Reply:
I spoke with @eldario and Michael Belobokov - there is no difference between `false` and `null` when we make a `make_report`-request.

Reply:
It does not matter. API calls must be same as previous

Reply:
Okay, I did a rollback 46fe77f2e35d1455319053d87569a45aaf03c717

Reply:
Now it seems much better! Thanks!